src/parser/binary-operation-parser.ts

```diff
@@ -100,6 +101,14 @@ export function parseFilterExpression(
   throw createFilterExpressionError(type, args)
 }
 
+export function parseWhereWithParametersAsLiterals(args: any[]): OperationNode {
+  if (args.length === 3) {
+    args = [args[0], args[1], sql.literal(args[2])]
```

Comment:
Instead of `sql.literal` you can use `ValueNode.createImmediate`.

Reply:
Good point! forgot this thing exists.